### DISCUSSION TITLE
Configurable proxy options for rewrites

### DISCUSSION THREAD

**Discussion by imZack** [vercel/next.js#37875] [Ideas]
### Describe the feature you'd like to request

Currently, there is no way to configure the proxy options for the "rewrites" function. For instance, if I want to disable [x-forward-x headers](https://github.com/vercel/next.js/pull/17557), it's hard coded with option `xfwd: true`.

### Describe the solution you'd like

I think making the `new Proxy({options})` configurable would be a way that users can easily customize the proxy's behavior.

### Describe alternatives you've considered

OR making the proxy instance replaceable (injecting from next.conf.js?)

**Comment by bbeck10** [comment]
I have a draft PR here - https://github.com/vercel/next.js/pull/40277

**Reply by lukeocodes** [reply]
@bbeck10 did this ever get resolved? this would have been extremely useful functionality
